File: Examples/DotNet/CSharp/PivotTables/GetWorksheetPivotTables.cs
using System;
using Com.Aspose.Cells.Api;
using Com.Aspose.Cells.Model;
using Com.Aspose.Storage.Api;

namespace PivotTables
{
    class GetWorksheetPivotTables
    {
        public static void Run()
        {
            // ExStart:1
            CellsApi cellsApi = new CellsApi(Common.APP_KEY, Common.APP_SID, Common.BASEPATH);
            StorageApi storageApi = new StorageApi(Common.APP_KEY, Common.APP_SID, Common.BASEPATH);

            String fileName = "Sample_Pivot_Table_Example.xls";
            String sheetName = "Sheet2";
            String storage = "";
            String folder = "";

            try
            {
                // Upload source file to aspose cloud storage
                storageApi.PutCreate(fileName, "", "", System.IO.File.ReadAllBytes(Common.GetDataDir() + fileName));

                // Invoke Aspose.Cells Cloud SDK API to get worksheet pivot table informations
                PivotTablesResponse apiResponse = cellsApi.GetWorksheetPivotTables(fileName, sheetName, storage, folder);

                if (apiResponse != null && apiResponse.Status.Equals("OK"))
                {
                    Console.WriteLine("Get Worksheet Pivot Tables Information, Done!");
                    Console.ReadKey();
                }
            }
            catch (Exception ex)
            {
                System.Diagnostics.Debug.WriteLine("error:" + ex.Message + "\n" + ex.StackTrace);
            }
            // ExEnd:1
        }
    }
}

File: Examples/PHP/Pivot Tables/GetWorksheetPivotTables.php
<?php
require_once realpath(__DIR__ . '/..') . '/vendor/autoload.php';
require_once realpath(__DIR__ . '/..') . '/Utils.php';

use Aspose\Cells\CellsApi;
use Aspose\Cells\AsposeApp;

class PivotTable {

	public $cells;

	public function __construct() {
		AsposeApp::$appSID = Utils::appSID;
		AsposeApp::$apiKey = Utils::apiKey;
		$this->cells = new CellsApi();
	}

	public function getWorksheetPivotTables() {
    	// Upload file to Aspose Cloud Storage
		$fileName = "Sample_Pivot_Table_Example.xls";
		Utils::uploadFile($fileName);

        $result = $this->cells->GetWorksheetPivotTables($name=$fileName, $sheetName="Sheet2", $storage = null, $folder = null);
        print_r($result);
	}
}

$pivotTable = new PivotTable();
$pivotTable->getWorksheetPivotTables();

?>